### PR TITLE
agda 2.6.2

### PR DESCRIPTION
**Pull request by haykam821**
Created with `brew bump-formula-pr`.

`resource` blocks may require updates.

### CODE REVIEW

File: Formula/agda.rb [hunk 1]
```diff
@@ -2,11 +2,10 @@ class Agda < Formula
...
-  revision 3
 
   stable do
-    url "https://hackage.haskell.org/package/Agda-2.6.1.3/Agda-2.6.1.3.tar.gz"
-    sha256 "bb1bc840cee95eea291bd220ea043b60153a6f7bd8198bc53be2bf3b83c8a1e7"
+    url "https://hackage.haskell.org/package/Agda-2.6.2/Agda-2.6.2.tar.gz"
+    sha256 "072caaad7a405b764423958d2001b68fbfd3fc8f98870cd0a5f0bcaeadfd3095"
 
     resource "stdlib" do
       url "https://github.com/agda/agda-stdlib/archive/v1.6.tar.gz"
```
Comment (carlocab) [inline Formula/agda.rb:11]: I think we can update this: https://github.com/agda/agda-stdlib/releases/tag/v1.7

Comment (haykam821) [reply Formula/agda.rb]: Fixed in https://github.com/Homebrew/homebrew-core/pull/79731/commits/9eae0c96c9b2d7c3a6a27a9118be41980684e36a

Comment (haykam821) [reply Formula/agda.rb]: Fixed in 9eae0c96c9b2d7c3a6a27a9118be41980684e36a

**Review by carlocab — APPROVED** [review]
Thanks, @haykam821.

**Pull request closed without merging**